client/src/panels/people/employee_prov.js

```diff
@@ -9,148 +9,18 @@ import {
-  constructor(){
-    super();
-  }
-  render() {
-    const { prov, years_by_province } = this.props;
-
-    const province_graph_title = function(prov){
-      if (prov === 'on' || prov === 'qc'){
-        prov += "lessncr";
-      }
-      return `${text_maker("five_year_history")} ${prov === "Canada" ? prov : provinces[prov].text}`;
-    };
-
-    const formatted_data = format_prov_data(prov,years_by_province);
-    const formatter = formats["big_int_raw"];
-    return (
-      <Fragment>
-        <p className="mrgn-bttm-0 mrgn-tp-0 nav-header centerer">
-          {province_graph_title(prov)}
-        </p>
-        <div style={{ height: "200px", width: "100%" }}>
-          <NivoResponsiveHBar
-            data = {formatted_data}
-            indexBy = "year"
-            keys = {["value"]}
-            enableLabel = {true}
-            label_format = { d=><tspan x={100} y={16}> {formatter(d)} </tspan>}
-            label={d => `${d.data.year}: ${formatter(d.value)}`}
-            colorBy ={d => get_graph_color(0.5)}
-            margin = {{
-              top: 40,
-              right: 30,
-              bottom: 20,
-              left: 20,
-            }}
-            padding = {0.1}
-            is_money = {false}
-            top_axis={{
-              tickSize: 5,
-              tickPadding: 5,
-              tickRotation: -30,
-              tickValues: 4,
-              format: (d) => formatter(d),
-            }}
-            remove_bottom_axis={true}
-            remove_left_axis={true}
-            add_top_axis={true}
-            enableGridX={false}
-            enableGridY={false}
-            isInteractive={false}
-          />
-        </div>
-      </Fragment>
-    );
-  }
-
-}
-
-
-class CanadaGraph extends React.Component {
-  constructor(){
-    super();
-    this.graph_area = React.createRef();
-  }
-  render() {
-    return <div ref={this.graph_area}/>;
-  }
-  componentDidMount() {
-    this._render();
-  }
-  _render() {
-    const { graph_args, prov_callback } = this.props;
-    const { years_by_province, color_scale } = graph_args;
-
-    const graph_area_sel = d3.select( ReactDOM.findDOMNode(this.graph_area.current) );
-
-    const formatter = formats["big_int_raw"];
-    const ticks = _.map(people_years, y => `${run_template(y)}`);
-    
-    const canada_graph = new Canada(graph_area_sel.node(), {
-      color: get_graph_color(1),
-      data: years_by_province,
-      ticks: ticks,
-      color_scale: color_scale,
-      formatter: formatter,
-    });
-
-    let active_prov = false;
-    canada_graph.dispatch.on('dataMouseEnter', prov => {
-      active_prov = true;
-      prov_callback(prov);    
-    });
-    canada_graph.dispatch.on('dataMouseLeave', () => {
-      _.delay(() => {
-        if (!active_prov) {
-          prov_callback("Canada");
-        }
-      }, 200);
-      active_prov = false;
-    });
-
-    canada_graph.render();
-  }
-}
+const context_years = people_years;
```

Comment:
`context_years` feels a lot less specific/informative than `people_years`. I'd prefer if `people_years` was used in this file (and only passed in to the chart config as `context_years`).